index.js

```diff
@@ -60,13 +62,9 @@ exports.handlers = {
 
       // Methods and hooks
       if (e.doclet.kind === 'function' && 'memberof' in e.doclet) {
-        if (e.doclet.memberof.endsWith('.methods')) {
-          e.doclet.scope = 'instance';
-          e.doclet.memberof = e.doclet.memberof.replace(/\.methods$/, ''); // force method to be displayed
-          e.doclet.meta.lineno += exportDefaultLines[fullPath] - mainDocletLines[fullPath];
-        } else {
```

Comment:
This is the line I'm referring to

Reply:
ooooh we don't want the hooks, ok. In my last commit I remove the test x)
Why don't we want them? Isn't it something that could be documented?
I would use documentation on hooks.

Reply:
Since they are hooks for internal Vue lifecycle, I think it's useless to document them.

What kind of documentation would you write for them? 🤔

Reply:
like i have these directives which handles a bunch of DOM setting in the mounted hook, and it would be cool to tel what it does.
There are the `vue-router` hooks also which are helpful (actually i didn't tested of it worked with vue-router hooks, but i suppose not) and would need some docs i think. But its just a personnal point of view i think.